tests/smoketest/smoketest.sh

```diff
@@ -97,6 +106,15 @@ echo "*** [INFO] Logs from snmp webhook..."
 oc logs "$(oc get pod -l app=default-snmp-webhook -o jsonpath='{.items[0].metadata.name}')"
 echo
 
+echo "*** [INFO] Cleanup resources..."
+oc delete job/stf-smoketest-${NAME}
```

Comment:
It would be nice to have an argument to skip this since I think it removes the failed pods that you might want to examine.

Reply:
That's a good idea

Reply:
Right, manual testing (or temporary no-cleanup on the CI server to permit examination) is what I'm thinking of. I often run this script manually after doing a deployment and would like to investigate if it fails.